EIPS/eip-1.md

```diff
@@ -252,6 +252,24 @@ Permitted Networking Specifications URLs must anchor to a specific commit, and s
 ^https://github.com/ethereum/devp2p/blob/[0-9a-f]{40}/.*$
 ```
 
+### Bitcoin Improvement Proposal
+
+Links to the Bitcoin Improvement Proposal may be included using normal markdown syntax, such as:
+
+```markdown
+[Bitcoin Improvement Proposal](https://github.com/bitcoin/bips)
+```
+
+Which renders to:
+
+[Bitcoin Improvement Proposal](https://github.com/bitcoin/bips)
```

Comment:
Please show the full link (so that it matches the regex.)

Reply:
Updated, thanks!